sql/org_mozilla_firefox_derived/migrated_clients_v1/init.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE
+  `moz-fx-data-shared-prod.org_mozilla_firefox_derived.migrated_clients_v1`
+PARTITION BY
+  (submission_date)
+CLUSTER BY
+  (fenix_client_id)
+AS
+WITH migrations AS (
+  SELECT
+    client_info.client_id AS fenix_client_id,
+    metrics.uuid.migration_telemetry_identifiers_fennec_client_id AS fennec_client_id,
+    submission_date,
+    normalized_channel,
+    versions.value AS value
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_firefox.migration`
+  LEFT JOIN
+    UNNEST(metrics.labeled_string.migration_migration_versions) versions
+  WHERE
+    submission_date
+    BETWEEN '2020-01-01'
+    AND current_date
+),
+clients AS (
+  SELECT
+    fenix_client_id,
+    MAX(submission_date) AS submission_date,
```

Comment:
```suggestion
    MIN(submission_date) AS submission_date,
```

We need to take the earliest migration date in order to have logical consistency between this and the incremental query.